.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
Please user `yarn` and do not ignore `lock` files.

Reply:
Ok, I will continue developing with yarn and fix this.